src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -258,4 +258,10 @@ protected String getHoldingId(Piece piece) {
   protected String getLocationId(Piece piece) {
     return "";
   }
+
+  @Override
+  protected String getReceivingTenantId(Piece piece) {
+    return StringUtils.EMPTY;
```

Comment:
getHoldingId and getLocationId also can be refactored to have the same code with StringUtils.EMPTY

Reply:
Then what is purpose of `locationId` and `holdingId` in class `ReceivedItem`?

Reply:
I mean
```suggestion
    @Override
  protected String getHoldingId(Piece piece) {
    return StringUtils.EMPTY;
  }
  @Override
  protected String getLocationId(Piece piece) {
    return StringUtils.EMPTY;
  }
```

Reply:
Got it, done